### PR TITLE
updated queries for lake

### PR DESCRIPTION
**Pull request by KV250025**
updated queries for external object store in vantage cloud lake usecases

### CODE REVIEW

File: VantageCloudLakeUseCases/content/ExternalObjectStore/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Open Editor to proceed with this use case.
 
 ### Introduction
 
-The following is a summary of how to access different formats of data stored in an object store. You can copy and modify the example queries below to access your own datasets. For simplicity the included datasets are setup to not need credentials, but it is highly recommended that you use credentials to access your own datasets.
+The following is a summary of how to copy data from VantageCloud Lake to an object store. You must provide your own bucket to execute the example queries below. We are using the __sample_sensor__ table to simulate copying data from a table in your vantage system to the external object store. You can copy and modify the example queries below to access your own datasets. For simplicity the included datasets are setup to not need credentials, but it is highly recommended that you use credentials to access your own datasets.
```
Comment (fs250044) [inline VantageCloudLakeUseCases/content/ExternalObjectStore/README.md:10]: I would simply this as:

This section shows how to copy data from VantageCloud Lake to an object store. To execute the example queries you must provide your own bucket. The **sample_sensor** table simulates copying data from a table in your Vantage system to the external object store. You can modify these examples to access to your own datasets. Although these example datasets do not require login credentials, you should use login credentials to access your own datasets.

File: VantageCloudLakeUseCases/content/ExternalObjectStore/README.md [hunk 2]
```diff
@@ -126,11 +145,11 @@ Before running the following examples, replace the following fields in the examp
 * [optionally] *AccessKey* : from the Access Key for your bucket - Secret Access Key example: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
 
 ### Example 1 
-This first example will select all rows in local sample_csv_local to copy the dataset to the object store's *sample1* partition:
+This first example will select all rows in sample_sensor to copy the dataset to the object store’s *sample1* partition:
```
Comment (fs250044) [inline VantageCloudLakeUseCases/content/ExternalObjectStore/README.md:148]: Simply as:

This example selects all rows in the **sample_sensor** table to copy the dataset to the object store's *sample1* partition:

**Review by fs250044 — COMMENTED** [review]
Comments added to simplify some of the text.

File: VantageCloudLakeUseCases/content/ExternalObjectStore/README.md (hunk 1 above)
Comment (fs250044) [reply VantageCloudLakeUseCases/content/ExternalObjectStore/README.md]: @KV250025: 

"... from a table in your vantage system to the external object store."

Is this for **any** Vantage system (in which case, simply capitalize "V"antage, or for VantageClould Lake only (in which case, write out the full product name).

**Review by halrlee — APPROVED** [review]

File: VantageCloudLakeUseCases/content/ExternalObjectStore/README.md (hunk 1 above)
Comment (KV250025) [reply VantageCloudLakeUseCases/content/ExternalObjectStore/README.md]: It is for any Vantage system. I used a upper case "V"antage in the changes https://github.com/Teradata/product-help/pull/58/files

**Review by vganji-td — APPROVED** [review]

**Pull request merged**